packages/expo-calendar/src/Calendar.ts

```diff
@@ -1122,6 +1251,8 @@ export async function getSourceAsync(id: string): Promise<Source> {
  * Sends an intent to open the specified event in the OS Calendar app.
  * @param id ID of the event to open.
  * @platform android
+ * @deprecated use `openEventInCalendarAsync` instead
```

Comment:
```suggestion
 * @deprecated Use [`openEventInCalendarAsync`](#openeventincalendarasyncparams-presentationoptions) instead.
```

Add link, switch into the sentence.